### PR TITLE
Preserve RSS article read status across updates

### PR DESCRIPTION
**Pull request by rjframe**
When a feed updates, the read status of articles are lost; this copies the read time to preserve its read status.

If an article body changed, this will not reset the article's status to unread. This could be done by comparing the update times first.

### CODE REVIEW

File: src/core/FeedsManager.cpp [hunk 1]
```diff
@@ -271,6 +271,8 @@ void Feed::update()
 
 										entry.publicationTime = normalizeTime(entry.publicationTime);
 
+										entry.lastReadTime = existingEntry.lastReadTime;
```
Comment (Emdek) [inline src/core/FeedsManager.cpp:274]: Please move this line just before `entry.publicationTime = normalizeTime(entry.publicationTime);`, without adding empty line.

**Review by Emdek — CHANGES_REQUESTED** [review]

**Comment by Emdek** [comment]
@rjframe, nice catch, but it needs small adjustment for coding style and stuff.

File: src/core/FeedsManager.cpp (hunk 1 above)
Comment (rjframe) [reply src/core/FeedsManager.cpp]: @Emdek This is fixed.

File: src/core/FeedsManager.cpp [hunk 2]
```diff
@@ -269,8 +269,10 @@ void Feed::update()
 											++amount;
 										}
 
+										entry.lastReadTime = existingEntry.lastReadTime;
 										entry.publicationTime = normalizeTime(entry.publicationTime);
 
+
```
Comment (Emdek) [inline src/core/FeedsManager.cpp:275]: Extra newline to be removed.

**Review by Emdek — CHANGES_REQUESTED** [review]

File: src/core/FeedsManager.cpp (hunk 2 above)
Comment (rjframe) [reply src/core/FeedsManager.cpp]: Ach. It's done.

**Pull request merged**